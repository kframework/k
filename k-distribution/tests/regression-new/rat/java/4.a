../haskell/4.a