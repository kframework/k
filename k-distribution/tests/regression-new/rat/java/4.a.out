../haskell/4.a.out